pkg/apiserver/rest/dualwriter.go

```diff
@@ -8,7 +8,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apiserver/pkg/registry/rest"
-	"k8s.io/klog/v2"
```

Comment:
this seems to be the log that's mostly used in grafana: https://github.com/grafana/grafana/tree/main/pkg/infra/log. What do you think of using that instead?

Reply:
Addressed similar comment in the other PR: https://github.com/grafana/grafana/pull/84605